packages/addons/addon-depends/system-tools-depends/i2c-tools/package.mk

```diff
@@ -3,10 +3,10 @@
 
 PKG_NAME="i2c-tools"
 PKG_VERSION="3.1.2"
-PKG_SHA256="f939a6242c03950cc568d0efdfa3db7a9c29e0e8f5abd01f2908bdd344c054ff"
+PKG_SHA256="6d6079153cd49a62d4addacef4c092db1a46ba60b2807070a3fbe050262aef87"
```

Comment:
there is a much more recent version, likely they include already the patches
https://mirrors.edge.kernel.org/pub/software/utils/i2c-tools/i2c-tools-4.2.tar.xz

Reply:
I had planned to bump lm_sensors and i2c-tools together.

https://i2c.wiki.kernel.org/index.php/I2C_Tools

```
i2c-tools                           | 3.1.2                                    | 4.2
lm_sensors                          | Commit date: 2018-11-23                  | Master: 2020-12-0
```